include/votca/tools/votca_tools_config.h.in

```diff
@@ -29,4 +29,8 @@
 
 /* Contact address of package */
 #define TOOLS_BUGREPORT "@PROJECT_CONTACT@"
+
+#define TOOLS_CMAKE_BINARY_DIR "@CMAKE_BINARY_DIR@"
```

Comment:
It is a really bad idea to put the build directory in the a public header. Just use:
```
target_compile_definitions(<test_executable> PRIVATE TOOLS_CMAKE_BINARY_DIR="${CMAKE_BINARY_DIR}")
```
instead.

Reply:
I like your solution a lot more. I can see why it is a bad idea, but why is it a really bad idea? Just want to learn

Reply:
Well, the build dir might be gone after install (or never existed for `rpm` installs), so we should really not have any paths referring to it in a public headers.

Reply:
good point, I removed it.